GliaWidgets/View/Chat/Entry/Upload/FileUploadListView.swift

```diff
@@ -4,23 +4,29 @@ import SwiftUI
 class FileUploadListView: UIView {
     var removeTapped: ((FileUpload) -> Void)?
 
+    var maxUnscrollableViews: Int {
+        if environment.uiApplication.preferredContentSizeCategory() <= .accessibilityMedium {
+            return 3
```

Comment:
Not having the context about this specific project, these numbers don't say much when reading them. I don't know if putting them into variables would solve the issue, so maybe having a comment for `maxUnscrollableViews` that says why 3 or 2 according to one or other case would be good.

Reply:
I support @gersonnoboa on this. I would recommend to add comments regarding the logic behind this computed property and the goal it achieves.

Reply:
7138828f0ce6e4548422b7865f271f1b79d9aa10